app/Http/Controllers/SettingsController.php

```diff
@@ -59,7 +59,8 @@ public function index()
         if (auth()->user()->me_contact_id) {
             $meContact = Contact::where('account_id', auth()->user()->account_id)
                 ->find(auth()->user()->me_contact_id);
-            $existingContacts->prepend($meContact);
+            if ($meContact)
+                $existingContacts->prepend($meContact);
```

Comment:
missing the brackets
```suggestion
            if ($meContact) {
                $existingContacts->prepend($meContact);
            }
```